lib/ModelMixins/CatalogMemberMixin.ts

```diff
@@ -182,3 +201,14 @@ namespace CatalogMemberMixin {
 }
 
 export default CatalogMemberMixin;
+
+/** Convenience function to get user readable name of a BaseModel */
+export function getName(model: BaseModel | undefined) {
+  return (
+    (CatalogMemberMixin.isMixedInto(model)
+      ? model.nameInCatalog ?? model.name
```

Comment:
Should this use `nameInCatalog`? The canonical name of the item is `model.name` and `nameInCatalog` should only be used in a context where the names of parent groups are also there, as it's intended that `nameInCatalog` is a short name to make catalog names less repetitive.

Reply:
ok no prob, i'll change it

Reply:
done